src/Controls/src/Core/Handlers/Items/iOS/SelectableItemsViewController.cs

```diff
@@ -55,7 +55,10 @@ internal void ClearSelection()
 
 			foreach (var index in selectedItemIndexes)
 			{
-				CollectionView.DeselectItem(index, true);
+				CollectionView.PerformBatchUpdates(null, _ =>
```

Comment:
Should we just move the entire foreach loop into "PerformBatchUpdates? 

Also. I think this code won't quite work if selectedItemIndexes has multiple values because the index will keep changing since it's not locally captured.

Reply:
good call, let me improve that . This was really needed, was more when it was set to null, but I wanted to fix all uses of `CollectionView.DeselectItem`

Reply:
I think this one still needs to be fixed yea?

The CV2 handler path was fixed but not this one.

Reply:
Yap ups!